packages/browser/src/integrations/onerror.ts

```diff
@@ -0,0 +1,33 @@
+import { Integration } from '@sentry/types';
+import { Raven } from '../raven';
+// @ts-ignore
+import { TraceKit } from './tracekit';
+
+/** Global OnError handler */
+export class OnError implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public name: string = 'OnError';
+  /**
+   * @inheritDoc
+   */
+  public handler: () => void = () => {
+    // tslint:disable-next-line:no-unsafe-any
+    Raven._handleOnErrorStackInfo.bind(Raven);
+  };
+  /**
+   * @inheritDoc
+   */
+  public install: () => void = () => {
```

Comment:
`install` & `uninstall` can be just `public install(): void {` it's shorter. `handler` must stay like this cause we pass it around.

Reply:
Can we just use an object in this case? There is no need for state.
If you don't like the syntax, you can also always `this.handle = this.handle.bind(this)` in the constructor.